custom/icds_reports/ucr/data_sources/dashboard/migrations_form.json

```diff
@@ -158,6 +158,46 @@
         },
         "is_nullable": true,
         "is_primary_key": false
+      },
+      {
+        "column_id": "is_migrated",
```

Comment:
I am assuming these new fields are tested that they working fine and are able to fetch the required data from form?
@ShubhamBansal1997